src/Gui/DlgExpressionInput.h

```diff
@@ -69,7 +102,16 @@ class GuiExport DlgExpressionInput : public QDialog
 
 public Q_SLOTS:
     void show();
-    void applyStylesheet();
+
+public Q_SIGNAL:
```

Comment:
This looks wrong. You should use `Q_SIGNALS` to mark the following group of functions as signals. `Q_SIGNAL` only marks the current line as a signal function.

On the other hand, I don't think you want signals here. What you need is a `WRITE` accessor, so that you'll know when someone has written to the property. `NOTIFY` is really for notifying other code that uses `DlgExpressionInput`.

Reply:
I replaced the signals by accessor functions, it does double the number of functions to add in the header and implement, but it is indeed a better solution.